service/docs/source/runtime.rst

```diff
@@ -3,13 +3,19 @@ Guide for Runtime Users
 =======================
 
 The GEOPM Runtime is software designed to enhance energy efficiency of
-applications though active hardware configuration.  The architecture is
+applications though active hardware configuration.  
+
+
+User Model
+----------
+
+The architecture is
 designed to provide a secure infrastructure to support a wide range
```

Comment:
Very minor nit: the line break is too early. Try to follow same column limit being followed in the rest of the document.